inc/package.class.php

```diff
@@ -471,12 +481,17 @@ private function parseApplication() {
       $manifest = $apk->getManifest();
       $iconResources = $apk->getResources($manifest->getApplication()->getIcon());
       $apkLabel = $apk->getResources($manifest->getApplication()->getLabel());
-      $input['icon'] = base64_encode(stream_get_contents($apk->getStream($iconResources[0])));
+      $stream = $apk->getStream($iconResources[0]);
+      if (!is_resource($stream)) {
+         return false;
```

Comment:
This is quite aggressive to fail when we cannot extract the icon. Instead I suggest to put a blank image. One transparent pixel with PNG should be OK. Hard codable in a constant and small.